test/jdk/java/nio/channels/DatagramChannel/StressNativeSignal.java

```diff
@@ -39,55 +45,78 @@ public class StressNativeSignal {
         try {
             serverSocketThread = new ServerSocketThread();
             serverSocketThread.start();
+        } catch (Exception z) {
+            System.err.println("failed to create and start a ServerSocketThread");
+            z.printStackTrace();
+        }
```

Comment:
It's not clear to me why there is a catch Exception here, same thing when creating/starting UDPThread, wouldn't it be simpler to have the constructor throw so that the test fails with the exception if there is an error starting either thread?

I assume the two fields can be final too.

Reply:
I will make the fields final.

I have retained the original structure of the test, and as indicated previously the tests are ambivalent to exceptions. I haven't changed that.  It would appears the test's objective is to trigger a 316 error and anything else is incidental. 

If there is a strong opinion in this regards, the test can be structured to fail fast  ... but note there are effectively two tests in one which can execute independent of each other to test the NativeThread.signal ... one or other might fail, but allow a valid test run.